game_event/views.py

```diff
@@ -150,3 +166,34 @@ def process_answer_game_event(request, id):
     content['event_exists'] = True
 
     return render(request, 'game_event/result.html', content)
+
+
+def get_weather_from_api(game_events):
```

Comment:
why this function doesn't just return the new list instead of running over again in the view function?

Reply:
@SchSeba I think that even though it is more code in the main view function `def game_events(request)` it is more readable if the final `game_events_with_weather` object that we send when rendering, is shown to the reader in this function, and not at the `def get_weather_from_api(game_events)` function.